mesa/visualization/templates/js/pltviz.js

```diff
@@ -0,0 +1,24 @@
+let PyPlotModule = function(canvas_width, canvas_height) {
```

Comment:
This should be `const`.

Reply:
And all the vars inside this function need to be declared with `const`/`let`. Default to `const` unless if it needs to be declared.

Reply:
`PyPlotModule` must be `const`.